ext/opentelemetry-ext-datadog/src/opentelemetry/ext/datadog/exporter.py

```diff
@@ -56,16 +56,26 @@ class DatadogSpanExporter(SpanExporter):
 
     Args:
         agent_url: The url of the Datadog Agent or use ``DD_TRACE_AGENT_URL`` environment variable
-        service: The service to be used for the application or use ``DD_SERVICE`` environment variable
+        service: The service name to be used for the application or use ``DD_SERVICE`` environment variable
+        env: Set the application’s environment or use ``DD_ENV`` environment variable
+        version: Set the application’s version or use ``DD_VERSION`` environment variable
+        tags: A list of default tags to be added to every span or use ``DD_TAGS`` environment variable
     """
 
-    def __init__(self, agent_url=None, service=None):
+    def __init__(
+        self, agent_url=None, service=None, env=None, version=None, tags=None
+    ):
         self.agent_url = (
             agent_url
             if agent_url
             else os.environ.get("DD_TRACE_AGENT_URL", DEFAULT_AGENT_URL)
         )
         self.service = service if service else os.environ.get("DD_SERVICE")
+        self.env = env if env else os.environ.get("DD_ENV")
```

Comment:
an empty string evaluates to false, so you may find that you cannot set that as a value (although I'm not sure what an empty string would do with regards to tag values anyway. It may not be valid).

also a shorthand for this is:
`self.env = env or os.environ.get("DD_ENV")` (evaluates to first non-falsey value)